src/components-examples/cdk/drag-drop/cdk-drag-drop-disabled/cdk-drag-drop-disabled-example.css

```diff
@@ -25,7 +25,7 @@
 
 .example-box.cdk-drag-disabled {
   background: #ccc;
-  cursor: default;
+  cursor: not-allowed;
```

Comment:
Might be good to also have `user-select: none` here since the original report also mentioned text selection.